<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinWidget</class>
 <widget class="QWidget" name="SkinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="topwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:transparent;background-image:url(:/img/skin/skin.png);}</string>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>皮肤中心</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/img/sysButton/close_button.png</normaloff>:/img/sysButton/close_button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bottomwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>800</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:3px; border:1px solid rgb(180, 190, 200); color:rgb(70, 70, 70); background:transparent;}
QWidget{background-color:white}
QPushButton:hover{color:rgb(100, 190, 250); border-radius:5px; border:1px solid rgb(0, 255, 255);}</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_4">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_5">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_6">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_7">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
     <property name="spacing">
      <number>60</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/4.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/5.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/6.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/7.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <property name="spacing">
      <number>60</number>
     </property>
     <item>
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/0.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qrc.qrc">:/img/skin/3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_0">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_1">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_2">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_3">
       <property name="text">
        <string>use skin</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
